### DISCUSSION TITLE
support for other devices/meters

### DISCUSSION THREAD

**Discussion by homecasagit** [wmbusmeters/wmbusmeters#611] [Q&A] [answered]
Hi, this looks like an amazing project I could really use. 

Does the list of supported meters mean that other meters will not work? Or ist it just a list of "confirmed working" devices? 

What would be needed to support other devices? 
I am especially talking about
  - Lorenz (water)
  - Engelmann SensoStar2/2+ (heating)

Maybe someone already tested these.  

Considering the current prices and availability of the dongles I would like to ask this prior to purchasing one just to find out it will not work.

Thanks in advance!

**Comment by weetmuts** [comment, marked as answer]
I am happy to add new devices as soon as you can provide a decrypted telegram! :-)

Supported means that there is a collected telegram in the regression-test suite, this will guarantee that the json is compatible in future releases.

You could buy a cheap rtlsdr dongle and run `wmbusmeters --logtelegrams rtlwmbus` to collect  some telegrams
that you can post here for analysis.

**Reply by homecasagit** [reply]
Perfect, thanks for the quick reply!

I will get the rtl-sdr.com V3, seems to be the best option currently available.

I will send you the telegrams once I collected them :)

**Comment by nico7576** [comment]
Hello
Than you for your huge work.
I'd like to know if my water metter is available ?
Ref. : DIEHL H21UA 193170 

For the moment, i've no hardware to collect any data since i don't know if it will be usefull.

Thank you verry much for your answer :)

![DIEHL](https://user-images.githubusercontent.com/24519700/192515057-5334ba49-5e51-43c7-aedc-1bbf33a9a0f2.jpg)

**Reply by weetmuts** [reply]
From the sticker it looks an oms meter addon which is supported. Driver is dme_07.
However you will probably need a decryption key. Buy a cheap rtlsdr and see if you can collect the telegrams.
Wmbusmeters will tell you if they are encrypted or not. 

Getting such a key can be difficult or easy. :-)

**Reply by nico7576** [reply]
Hi. FYI, i do not need any decryption key.
This command :
`wmbusmeters --logtelegrams --format=json auto:t1 watermeter dme_07 myID NOKEY`
returns 
`{"media":"water","meter":"dme_07","name":"watermeter","id":"myID","status":"OK","total_m3":65.257,"timestamp":"2022-10-14T08:32:40Z","device":"rtlwmbus[00000001]","rssi_dbm":103}`

Thank you for your previous help 🥇
